tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -11640,7 +11640,7 @@ TEST_F(SQLiteFixture, DOASDirectToZone_ZoneMultiplierRemoved)
                             "  AND RowName = 'DOAS Direct to Zone'");
     // check the value from result records
     Real64 return_val_total = execAndReturnFirstDouble(query_total);
-    EXPECT_EQ(return_val_total, 515.19);
+    EXPECT_EQ(return_val_total, 598.2);
```

Comment:
The change is to account for standard air density in the DOAS load calculation. Example `515.19  * 1.161125 = 598.20`.